cylc/flow/cfgspec/globalcfg.py

```diff
@@ -1050,55 +1048,22 @@ def default_for(
 
                     .. versionadded:: 8.0.0
                 """)
-
     with Conf('editors', desc='''
-        Choose your favourite text editor for editing workflow configurations.
-    '''):
-        Conf('terminal', VDR.V_STRING, desc='''
-            An in-terminal text editor to be used by the Cylc command line.
-
-            If unspecified Cylc will use the environment variable
-            ``$EDITOR`` which is the preferred way to set your text editor.
+        Configured text editors.
 
-            .. Note::
+        .. deprecated:: 8.0.0
 
-                You can set your ``$EDITOR`` in your shell profile file
-                (e.g. ``~.bashrc``)
-
-            If neither this or ``$EDITOR`` are specified then Cylc will
-            default to ``vi``.
-
-            Examples::
-
-               ed
-               emacs -nw
-               nano
-               vi
+           Cylc no longer needs a configured text editor: ``cylc edit`` has
+           been removed. ``cylc trigger --edit`` will be reimplimented
+           differently in a future release. To view workflow logs, open them
+           directly or send ``cylc cat-log`` output to a file or editor.
+    '''):
+        Conf('terminal', VDR.V_STRING, desc='''
+            Terminal text editor (deprecated).
         ''')
         Conf('gui', VDR.V_STRING, desc='''
-            A graphical text editor to be used by cylc.
-
-            If unspecified Cylc will use the environment variable
-            ``$GEDITOR`` which is the preferred way to set your text editor.
-
-            .. Note::
-
-               You can set your ``$GEDITOR`` in your shell profile file
-               (e.g. ``~.bashrc``)
-
-            If neither this or ``$GEDITOR`` are specified then Cylc will
-            default to ``gvim -fg``.
-
-            Examples::
-
-               atom --wait
-               code --new-window --wait
-               emacs
-               gedit -s
-               gvim -fg
-               nedit
+            Graphical text editor (deprecated).
```

Comment:
I thought this was getting removed & obsoleted?

Reply:
It was, then @wxtim suggested this:
> Asking for addition of obsoletion messages so that anyone with an older config gets a useful message.

... and we both temporarily forgot that we're not doing that for the global config :rofl: